src/generator.ts

```diff
@@ -132,7 +147,8 @@ export class Generator {
         type: 'confirm',
         name: 'installPlaywrightBrowsers',
         message: `Install Playwright browsers (can be done manually via '${this.packageManager.npx('playwright', 'install')}')?`,
-        initial: true,
+        initial: true,    
+        skip: !!this.options.browsers,
```

Comment:
skip the prompt question, defaulting to `true`, by using the `enquirer` prompt option `skip`:

- https://github.com/enquirer/enquirer#prompt-options